spec/features/visitors/set_password_spec.rb

```diff
@@ -17,6 +17,10 @@
       confirm_last_user
     end
 
+    it 'does not have the password strength bar' do
+      expect(page).not_to have_content(t('instructions.password.strength.intro'))
+    end
+
```

Comment:
JavaScript specs are very slow to run, so I'd try to consolidate them as much as possible. Since this context appears to be largely concerned with the behavior of the empty field validation, I don't think we need to assert about the password strength meter.

```suggestion
```